tests/dummy/app/demo/slots-pickers/template.hbs

```diff
@@ -1,5 +1,6 @@
 <h1>slots pickers</h1>
 
+{{link-to "easy" "demo.slots-pickers.easy-slot-picker" class="m3 btn btn-primary"}}
```

Comment:
what is 'easy'?

Reply:
Easy, because that's the simplest way to consume the calendars - just drop one component.but it's not customizable. We are on the slots-picker page so just showing the single names.